src/ai-bundle/src/AIBundle.php

```diff
@@ -158,11 +157,12 @@ private function processPlatformConfig(string $type, array $platform, ContainerB
             $platformId = 'ai.platform.anthropic';
             $definition = (new Definition(Platform::class))
                 ->setFactory(AnthropicPlatformFactory::class.'::create')
-                ->setAutowired(true)
                 ->setLazy(true)
                 ->addTag('proxy', ['interface' => PlatformInterface::class])
                 ->setArguments([
                     '$apiKey' => $platform['api_key'],
+                    '$httpClient' => new Reference('http_client', ContainerInterface::NULL_ON_INVALID_REFERENCE),
+                    '$contract' => new Reference('ai.platform.contract'),
```

Comment:
Just injecting the default contract here, will disable the `AnthropicContract` and will cause errors I guess, see

https://github.com/symfony/ai/blob/1d5dabb6356a6fa00015a543f05bc250247fc6a2/src/platform/src/Bridge/Anthropic/PlatformFactory.php#L30-L37

Reply:
potentially same for other places

Reply:
I'd still prefer getting the instance from DI, so I'll add anthropic platform as separate service